src/overlays/actors/ovl_En_Cow/z_en_cow.h

```diff
@@ -3,21 +3,37 @@
 
 #include "ultra64.h"
 #include "global.h"
+#include "assets/objects/object_cow/object_cow.h"
 
+#define EN_COW_TYPE(thisx) ((thisx)->actor.params)
+
+#define EN_COW_FLAG_PLAYER_HAS_APPROACHED (1 << 1)
+#define EN_COW_FLAG_WONT_GIVE_MILK (1 << 2)
+
+typedef enum {
+    /*  0 */ EN_COW_TYPE_DEFAULT,
+    /*  1 */ EN_COW_TYPE_TAIL
+} EnCowType;
+
+typedef enum {
+    /*  0 */ EN_COW_COLL_FRONT,
+    /*  1 */ EN_COW_COLL_REAR,
+    /*  2 */ EN_COW_COLL_MAX
+} EnCowCollider;
```

Comment:
I would spell `COLLIDER` in full instead of `COLL`
there are issues with disambiguation between colliders (z_collision_check) and surface collision (z_bgcheck) so avoiding confusion where possible